packages/sysweb3-keyring/src/transactions/ethereum.ts

```diff
@@ -110,7 +110,7 @@ export const EthereumTransactions = (): IEthereumTransactions => {
       const resp = concatSig(toBuffer(sig.v), sig.r, sig.s);
       return resp;
     } catch (error) {
-      throw error;
+      throw new Error(error);
```

Comment:
In try-catch blocks, was defined to use `throw error` instead `throw new Error(err)` cause we will handle error inside pali, and we need to send error without be instance of Error.